CONFIGURATION.md

```diff
@@ -195,12 +195,14 @@ regexp: <regex>,
 
 # The query sent in the TCP probe and the expected associated response.
 # "expect" matches a regular expression;
+# "expect_bytes" does exact byte-by-byte match.
+# "expect" and "expect_bytes" are mutually exclusive.
```

Comment:
let's move this to line above, so each line is for one a config param.


I am thinking something like `# "expect_bytes" does exact byte-by-byte match; "expect" and "expect_bytes" are mutually exclusive.`